Ansible/collection/jfrog/ansible/galaxy.yml

```diff
@@ -45,13 +48,13 @@ tags: []
 dependencies: {}
 
 # The URL of the originating SCM repository
-repository: http://example.com/repository
+repository: "https://github.com/jfrog/JFrog-Cloud-Installers/"
 
 # The URL to any online docs
-documentation: http://docs.example.com
+documentation: "https://www.jfrog.com/confluence/display/JFROG/"
 
 # The URL to the homepage of the collection/project
-homepage: http://example.com
+homepage: "https://github.com/jfrog/JFrog-Cloud-Installers/"
```

Comment:
I believe this should point to https://github.com/jfrog/JFrog-Cloud-Installers/tree/master/Ansible

Reply:
Done.